pandas/tests/io/test_pytables.py

```diff
@@ -2920,6 +2920,33 @@ def test_store_index_name_with_tz(self):
             recons = store['frame']
             tm.assert_frame_equal(recons, df)
 
+    def test_store_index_name_numpy_str(self):
+        # GH #13492
+        idx = pd.Index(pd.to_datetime([datetime.date(2000, 1, 1),
+                                       datetime.date(2000, 1, 2)]),
+                       name=u('cols\u05d2'))
+        idx1 = pd.Index(pd.to_datetime([datetime.date(2010, 1, 1),
+                                        datetime.date(2010, 1, 2)]),
+                       name=u('rows\u05d0'))
+        df = pd.DataFrame(np.arange(4).reshape(2, 2), columns=idx, index=idx1)
+
+        # This used to fail, returning numpy strings instead of python strings.
+        with ensure_clean_path(self.path) as path:
+            df.to_hdf(path, 'df', format='fixed')
+            df2 = read_hdf(path, 'df')
+            assert_frame_equal(df, df2)
+
+            assert type(df2.index.name) == str
+            assert type(df2.columns.name) == str
+
+        with ensure_clean_path(self.path) as path:
+            df.to_hdf(path, 'df', format='table')
```

Comment:
You can parametrize `format` I think

@pytest.mark.parametrized('format', ['table', 'fixed'])
def test_store_index_name_numpy_str(self, format):
    ...

```